### PR TITLE
Rely on latest in the install instructions.

### PR DESCRIPTION
**Pull request by rgbkrk**
This switches us over to relying on `latest` for install.

/cc @ktbartholomew and @carolynvs for review!

### CODE REVIEW

**Comment by rgbkrk** [comment]
As chosen by @ktbartholomew, our next release will be titled "Classy Cassiopeia".

File: README.md [hunk 1]
```diff
@@ -19,33 +19,26 @@ If you're using `brew`:
...
-$ chmod u+x ~/bin/carina
-```
-
-### Linux
-
-There are links to built binaries over in [releases](https://github.com/getcarina/carina/releases).
-
-After downloading the version for your system, you'll probably need to rename it,
-set it as executable, and put it on a `PATH` you have:
+Download a copy of `carina` straight from latest or via the links in [releases](https://github.com/getcarina/carina/releases), move it somewhere on your `$PATH`, and make it executable:
```
Comment (carolynvs) [inline README.md]: We are saying "straight from latest" but not linking to the latest url.

File: README.md [hunk 2]
```diff
@@ -19,33 +19,26 @@ If you're using `brew`:
...
 
-There are links to built binaries over in [releases](https://github.com/getcarina/carina/releases). It runs as a console client, so open up your favorite PowerShell or cmd.exe and run `carina`:
+There are links to built binaries over in [releases](https://github.com/getcarina/carina/releases). For a quick install, open up Powershell and run:
 
 ```
-> carina.exe
+> iex (wget https://download.getcarina.com/carina/latest/Windows/x86_64/carina)
 ```
 
+You'll want to move carina to somewhere on your `%PATH%`.
```
Comment (carolynvs) [inline README.md]: "You'll want to move carina.exe..."

File: README.md [hunk 3]
```diff
@@ -19,33 +19,26 @@ If you're using `brew`:
...
 ```
 
 ### Windows
 
-There are links to built binaries over in [releases](https://github.com/getcarina/carina/releases). It runs as a console client, so open up your favorite PowerShell or cmd.exe and run `carina`:
+There are links to built binaries over in [releases](https://github.com/getcarina/carina/releases). For a quick install, open up Powershell and run:
 
 ```
-> carina.exe
+> iex (wget https://download.getcarina.com/carina/latest/Windows/x86_64/carina)
```
Comment (carolynvs) [inline README.md]: This should be `wget https://download.getcarina.com/carina/latest/Windows/x86_64/carina.exe -OutFile carina.exe`. 

FYI, `iex` in PowerShell is the equivalent of `foo | sh`.

File: README.md (hunk 1 above)
Comment (rgbkrk) [inline README.md]: I need someway to convey that we'll use latest in the following command. Wonder what I should change this to.

File: README.md (hunk 3 above)
Comment (rgbkrk) [inline README.md]: My dvm copy pasta with no context was so bad I'm embarrassed.

File: README.md (hunk 1 above)
Comment (carolynvs) [inline README.md]: Doh! That's my bad, I was reviewing in diff mode and missed the command below. Carry on! :+1:

**Comment by rgbkrk** [comment]
Comments addressed!

**Pull request merged**